integration_tests/commands/async/hmget_test.go

```diff
@@ -0,0 +1,101 @@
+package async
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestHMGET(t *testing.T) {
```

Comment:
To reduce any chances of flakiness, defer a method which cleans up any keys created by this test at the end of execution.

Reply:
Hey, there are defer methods that close the connection as well as clear the keys - On line 34 and 35, do I move them up so that it's more clear?

Reply:
sure